### DISCUSSION TITLE
Help with Video Downloadhelper license key

### DISCUSSION THREAD

**Discussion by kaicyee** [aclap-dev/video-downloadhelper#1036] [Help and Q&A]
### Your question

I've been a Video Downloadhelper paid member for many years. Recently I had a computer crash and it wiped out the license key in my Video Downloadhelper Firefox extension. I tried  Recover License Key link but I get no email with new License key. I know Video Downloadhelper has me on record because they didn't reject the email address. Is there another way to Recover License Key? I really don't want to buy it yet again.

### Addon & CoApp version and details

Video Downloadhelper 9.0.2.12
CoApp 2.0.19.0

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Since you apparently found the license recovery link, I'm assuming you got it from the FAQ entry in the Wiki.  That entry contains further instructions.  Read it all.  There's an E-mail address in there that you should communicate with.
